src/main/scala/is/hail/expr/Relational.scala

```diff
@@ -919,23 +933,11 @@ case class MatrixMapRows(child: MatrixIR, newRow: IR) extends MatrixIR {
 
         val entriesOff = localRowType.loadField(region, oldRow, entriesIdx)
 
-        val aggResultsOff = if (seqOps.nonEmpty) {
+        val aggResultsOff = if (rvAggs.nonEmpty) {
           val newRVAggs = rvAggs.map(_.copy())
 
-          var i = 0
-          while (i < localNCols) {
-            val eMissing = localEntriesType.isElementMissing(region, entriesOff, i)
-            val eOff = localEntriesType.elementOffset(entriesOff, localNCols, i)
-            val colMissing = localColsType.isElementMissing(region, cols, i)
-            val colOff = localColsType.elementOffset(cols, localNCols, i)
+          seqOps()(region, newRVAggs, oldRow, false, globals, false, cols, false, oldRow, false)
```

Comment:
I'm not *entirely* sure what's happening here, but I think the TAGG/AggIn argument here is wrong too---see #3434